setup.py

```diff
@@ -57,7 +57,7 @@ def run(self):
 
 
 setup(name='citrine',
-      version='0.71.0',
+      version='0.71.1',
```

Comment:
I think a new option should be a minor version bump.

Reply:
Yeah good point.